functional-test/src/test/groovy/org/rundeck/util/container/BaseContainer.groovy

```diff
@@ -186,6 +186,25 @@ abstract class BaseContainer extends Specification implements ClientProvider {
         }
     }
 
+    boolean waitExecution(int executionId) {
```

Comment:
theres already a method in JobUtils.groovy to wait for an execution to be XXXX state

Reply:
The test will only wait a while for the job to be executed, I replace it with a waiting time of 5 seconds

Reply:
it looks like this method is not being used